AeroGearPush-Sw.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name         = "AeroGearPush-Sw"
-  s.version      = "3.0.7"
+  s.version      = "3.0.1"
   s.summary      = "AeroGear UnifiedPush Client Registration SDK (Swift)."
-  s.homepage     = "https://github.com/coco-jam/aerogear-ios-push"
+  s.homepage     = "https://github.com/aerogear/aerogear-ios-push/tree/Swift4.2"
   s.license      = 'Apache License, Version 2.0'
   s.author       = "Red Hat, Inc."
-  s.source       = { :git => 'https://github.com/coco-jam/aerogear-ios-push.git', :tag => s.version }
+  s.source       = { :git => 'https://github.com/aerogear/aerogear-ios-push.git', :tag => s.version }
```

Comment:
Should this also be specifying the `Swift4.2` branch instead of a tag? As no tag is bring created for these changes.

Reply:
@davidffrench Yes and no :) To be very honest we don't need tag or branch, only when going to release it for cocoapods, which (I think) is not the case. Developer can use it from a branch if he needs

He should use
```
pod 'AeroGearPush-Swift', :git => 'https://github.com/aerogear/aerogear-ios-push.git', :branch => 'Swift4.2'
```

instead of 

```
pod 'AeroGearPush-Swift
```

Reply:
:+1: